clang/unittests/Basic/FileManagerTest.cpp

```diff
@@ -284,7 +284,6 @@ TEST_F(FileManagerTest, getFileRefReturnsCorrectNameForDifferentStatPath) {
   ASSERT_FALSE(!F1Alias);
   ASSERT_FALSE(!F1Alias2);
   EXPECT_EQ("dir/f1.cpp", F1->getName());
-  EXPECT_EQ("dir/f1.cpp", F1->getFileEntry().getName());
```

Comment:
Maybe should keep this until the function is removed and use pramga to disable the warning here?

Reply:
I created `LLVM_IGNORE_DEPRECATIONS_OF_DECLARATIONS_{BEGIN,END}`. Let me know if you think this makes sense to keep in LLVM's Support library, or would make more local.